src/SprykerSdk/Shared/SprykGui/SprykGuiConstants.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Shared\SprykGui;
+
+/**
+ * Declares global environment configuration keys. Do not use it for other class constants.
+ */
+interface SprykGuiConstants
+{
+    public const ENABLE_GUI = 'SPRYK_GUI:ENABLE_GUI';
```

Comment:
why is this env const? this should be only config imo.

Reply:
Hi, I have removed it from env const and just use it in the config.